src/domains/job/recruitment/detail/page/RecruitmentDetailPage.jsx

```
@@ -45,14 +48,26 @@ const RecruitmentDetailPage = () => {
     }
   }
 
+  const handleScrapClick = async () => {
+    try {
+      await toggleScrap(id, 'JOB_POSTING')
+    } catch (e) {
```

ESLint

[error] 'e' is defined but never used.

(no-unused-vars)